Gemfile

```diff
@@ -18,36 +18,23 @@ def location_for(place_or_version, fake_version = nil)
 end
 
 group :development do
-  gem 'puppet', *location_for(ENV['PUPPET_GEM_VERSION'])
+  gem 'openvox', *location_for(ENV['PUPPET_GEM_VERSION'])
 
   gem 'json_spec', '~> 1.1', '>= 1.1.5'
 
   gem 'mdl', '~> 0.11.0'
 
-  gem 'pry', require: false
-  gem 'pry-byebug', require: false
-  gem 'pry-stack_explorer', require: false
-
   # Need the following otherwise we end up with puppetlabs_spec_helper 1.1.1
   gem 'mocha', '~> 1.0'
-  gem 'puppetlabs_spec_helper'
 
   gem 'rake'
   gem 'rspec', '~> 3.1'
   gem 'rspec-its', '~> 1.0'
 
-  gem 'rubocop', '~> 1.70.0', require: false
-  gem 'rubocop-performance', '~> 1.16', require: false
-  gem 'rubocop-rspec', '~> 3.0', require: false
-
-  gem 'serverspec'
-  gem 'simplecov', require: false
-  gem 'simplecov-console', require: false
-
   gem 'redcarpet'
 end
 
-group :acceptance do
-  gem 'puppet_litmus'
-  gem 'net-ssh'
-end
+# puppet/openvox on Ruby 3.3 / 3.4 has some missing dependencies
```

Comment:
Let's fix those in openvox now we can

Reply:
removed those workarounds except for syslog, thats a bit more tricky